server/server.js

```diff
@@ -158,7 +159,10 @@ async function initKubeClient() {
     const conf =
       process.env.NODE_ENV === 'production' ? Request.config.getInCluster() : Request.config.fromKubeconfig();
     const backend = new Request(conf);
-    const kubeclient = new Client({ backend });
+    backend.requestOptions.insecureSkipTlsVerify = true;
+    backend.requestOptions.strictSSL = false;
```

Comment:
these are just to make MDC work with a remote openshift using selfsigned certs. Will be removed before merging the PR.